src/content/app/tools/blast/state/blast-results/blastResultsSlice.ts

```diff
@@ -68,6 +68,9 @@ export type BlastSubmission = {
     status: JobStatus;
     data: null; // TODO: add data type
   }>;
+  ui: {
+    isExpandedOnSubmissionList: boolean; // used to retain expanded / collapsed state in the jobs list view
+  };
```

Comment:
I am getting this error now — I suppose because the submission stored in my browser's storage doesn't have the `ui` property.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/6834224/181711886-c7301d32-ffce-488c-96a1-dece9cf9508c.png">

I was kinda hoping that this `ui` property would live independently from the `BlastSubmission` type:

```ts
type BlastResultsState = {
  submissions: Record<string, BlastSubmission>;
  ui: {
    unviewedJobsPage: {
      expandedSubmissionIds: string[];
    },
    viewedJobsPage: {
      expandedSubmissionIds: string[];
    },
    submissionDetailsPage: { // or submissionResultsPage
      ...
    }
  }
}
```